src/lib.rs

```diff
@@ -1,53 +1,117 @@
+//! Inquirer – Fancy user interaction on the command line
+
+// Termion depends on nighty (for Keys), we need this for the unstable `std::io::CharsError` 
```

Comment:
Yep, I'll fix it tonighty!

Philip Munksgaard <notifications@github.com> schrieb am Do. 14. Juli 2016
um 23:07:

> In src/lib.rs
> <https://github.com/Munksgaard/inquirer-rs/pull/1#discussion_r70884114>:
>
> > @@ -1,53 +1,117 @@
> > +//! Inquirer – Fancy user interaction on the command line
> > +
> > +// Termion depends on nighty (for Keys), we need this for the unstable `std::io::CharsError`
>
> I believe nighty should probably be nightly
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Munksgaard/inquirer-rs/pull/1/files/3d7279489e978d7acdfe9b21d616a4f6962b02a8#r70884114>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AABOX-h5VUFKOKvdYPO43JbwpohojQPsks5qVqUggaJpZM4JMcPZ>
> .
>